OpenRA.Mods.Common/Widgets/Logic/Installation/InstallFromDiscLogic.cs

```diff
@@ -270,7 +270,7 @@ void InstallFromDisc(string path, ModContent.ModSource modSource)
 									var sourceDir = Path.Combine(path, i.Value.Value);
 									foreach (var node in i.Value.Nodes)
 									{
-										var sourcePath = Path.Combine(sourceDir, node.Value.Value);
+										var sourcePath = OpenRA.FileSystem.FileSystem.ResolveCaseInsensitivePath(Path.Combine(sourceDir, node.Value.Value));
```

Comment:
Can we please define `using FS = OpenRA.FileSystem.FileSystem;` and then use `FS.ResolveCaseInsensitivePath` for consistency with other places where we use `FileSystem`?